### PR TITLE
productize _gen_mac_for_libvirt()

### PR DESCRIPTION
**Pull request by rplevka**
generate a standalone wrapper function since the gen_mac is being used on multiple places across the Class

```python
In [4]: host = LibvirtGuest(extra_nic = True)

In [5]: host.create()
2017-01-17 15:50:58 - robottelo.ssh - INFO - Instantiated Paramiko client 0x7f3c13fe5750
2017-01-17 15:50:58 - robottelo.ssh - DEBUG - Connected to [libvirt1.com]
2017-01-17 15:50:58 - robottelo.ssh - DEBUG - >>> virt-install --hvm --network=bridge:downstream_el6 --mac=8e:22:2c:9f:15:76 --name=mac8e222c9f1576.satellite.com --ram=1024 --vcpus=1 --os-type=linux --os-variant=rhel7 --disk path=/var/lib/libvirt/images/mac8e222c9f1576.satellite.com.img,size=8 --noautoconsole --pxe --network=bridge:downstream_el6 --mac=c6:3b:39:5c:a7:5b
2017-01-17 15:51:01 - robottelo.ssh - DEBUG - <<< stdout

Starting install...
Allocating 'mac8e222c9f1576.satellite.c | 8.0 GB  00:00     
Creating domain...                                          |    0 B  00:00     
Domain installation still in progress. You can reconnect to 
the console to complete the installation process.

2017-01-17 15:51:01 - robottelo.ssh - INFO - Destroying Paramiko client 0x7f3c13fe5750
2017-01-17 15:51:01 - robottelo.ssh - INFO - Destroyed Paramiko client 0x7f3c13fe5750

In [6]: host.attach_nic()
2017-01-17 15:51:10 - robottelo.ssh - INFO - Instantiated Paramiko client 0x7f3c13fe5750
2017-01-17 15:51:10 - robottelo.ssh - DEBUG - Connected to [libvirt1.com]
2017-01-17 15:51:10 - robottelo.ssh - DEBUG - >>> virsh attach-interface --domain=mac8e222c9f1576.satellite.com --type=bridge --source=downstream_el6 --model=virtio --mac=8e:4b:d6:3d:c2:12 --config --live
2017-01-17 15:51:11 - robottelo.ssh - DEBUG - <<< stdout
Interface attached successfully


2017-01-17 15:51:11 - robottelo.ssh - INFO - Destroying Paramiko client 0x7f3c13fe5750
2017-01-17 15:51:11 - robottelo.ssh - INFO - Destroyed Paramiko client 0x7f3c13fe5750

```

### CODE REVIEW

**Comment by sghai** [comment]
ACK

File: robottelo/libvirt_discovery.py [hunk 1]
```diff
@@ -18,6 +18,17 @@
 logger = logging.getLogger(__name__)
 
 
+def _gen_mac_for_libvirt():
+    # fe:* MAC range is considered reserved in libvirt
+    for _ in range(0, 10):
+        mac = gen_mac(multicast=False, locally=True)
+        if not mac.startswith(u'fe'):
+            return(mac)
```
Comment (rochacbruno) [inline robottelo/libvirt_discovery.py]: We missed this one in review ....

Comment (rochacbruno) [inline robottelo/libvirt_discovery.py]: `return mac`

Comment (rochacbruno) [inline robottelo/libvirt_discovery.py]: @rplevka we may need a new PR to fix that "(mac)" in master...

**Pull request merged**